sphinx/activity/call-activity/src/main/java/com/example/call_activity/CallActivity.kt

```diff
@@ -163,95 +165,44 @@ class CallActivity : AppCompatActivity() {
             }
         }
 
-        lifecycleScope.launch {
-            viewModel.startRecordingState
-                .flowWithLifecycle(lifecycle, Lifecycle.State.STARTED)
-                .collectLatest {
-                    (recordingState: StartRecordingState, isLocalParticipantRecording: Boolean)  ->
-
-                when (recordingState) {
-                    StartRecordingState.Error -> {
-                        binding.recordButton.isEnabled = true
-                        showCustomToastMessage(messageRes = R.string.error_message_call_recording)
-                    }
-
-                    StartRecordingState.Empty -> {}
-
-                    StartRecordingState.Recording -> {
-                        showCustomToastMessage(messageRes = R.string.call_recording_in_progress_message)
-
-                        binding.apply {
-                            recordButton.also {
-                                it.setImageDrawable(getRecordDrawable(isLocalParticipantRecording))
-                                it.isEnabled = isLocalParticipantRecording
-                            }
-
-                            fadeInFadeOutAnimation(
-                                viewToFadeOut = recordButton,
-                                durationMillis = 700,
-                                alphaStartValue = 1.0f,
-                                alphaEndValue = 0.5f,
-                            )
-                        }
-                    }
-
-                    StartRecordingState.Loading -> {
-                        showCustomToastMessage(messageRes = R.string.starting_call_recording_message)
-
-                        binding.recordButton.isEnabled = false
-                    }
-                }
```

Comment:
This portion of code you are removing is related with call recording and shouldn't be removed